RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -106,18 +106,30 @@ import RevenueCat
 
     func loadHasSubscriptions() async {
```

Comment:
I know this method is becoming crazy and I am planning on refactoring, but I need to do it step by step otherwise it becomes very complex. If we add subscription information to the CustomerInfo this will be easier